pages/platform/private-registry/configure-oidc-provider-authentication/meta.yaml

```diff
@@ -0,0 +1,2 @@
+id: f8f9ecd0-be1c-4bd3-ab58-25599446ea2e
+full_slug: public-cloud-private-registry-sign-artifacts-with-cosign
```

Comment:
Slug looks like a bad copy paste ?

Reply:
yes, writing docs with 32degree at home is not good :-D